docs/sources/enterprise/access-control/_index.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Fine-grained access control"
+description = "Understand how to grant, change or revoke access to the Grafana resources"
+keywords = ["grafana", "fine-grained-access-control", "roles", "permissions", "enterprise"]
+weight = 100
++++
+
+# Fine-grained access control
+
+> **Note:** Fine-grained access control is currently in beta. Expect changes in future releases.
```

Comment:
```suggestion
> **Note:** Fine-grained access control is in beta, and you can expect changes in future releases.
```

Reply:
One day, would be nice to update these in other docs. Since I suspect many people, including me are just looking at the previous examples and copy pasting it.